EnglishFilter/sections/specific.txt

```diff
@@ -117,6 +117,7 @@ erowall.com,adultwalls.com,motaen.com,camclips.cc,camwhores.biz,freeporn-site.co
 ! TotemTools
 /\/t[a-f0-9]{10,13}\.min\.js$/$domain=nudevideoswallpapers.com
 !#%#//scriptlet("abort-on-property-read", "_TotemTools")
+freshscat.com#%#//scriptlet("abort-on-property-read", "puShown", "/doOpen|popundr/")
```

Comment:
I'm not sure, but I think that this rule is incorrect and as I understand `abort-on-property-read` requires only one parameter - https://github.com/AdguardTeam/Scriptlets#-abort-on-property-read
It seems that it works, but this part `"/doOpen|popundr/"` is not needed (and probably do nothing) or maybe it should be `abort-current-inline-script`?

Anyway, scriptlets are not supported by apps yet, so I guess that it would be better to add this website to this rule
https://github.com/AdguardTeam/AdguardFilters/blob/b159a2243646c7bdec5da20a02ecb7328982dfbd/EnglishFilter/sections/general_extensions.txt#L103